services/ffc/ffc-sfd/snd/03/kustomization.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-resources:             
-  - ../../base
-  - ../../ffc-sfd-permissions
-  - ../../ffc-sfd-ahwp-stub
-  - ../../ffc-sfd-auth
-  - ../../ffc-sfd-messages
-  - ../../ffc-sfd-messages-processor
-  - ../../ffc-sfd-landing-page
-  - ../../ffc-sfd-data
-  - ../../ffc-sfd-permissions-api
-  - ../../ffc-sfd-business  
+resources:
+- ../../base
+- ../../ffc-sfd-business
+- ../../ffc-sfd-permissions
+- ../../ffc-sfd-landing-page
+- ../../ffc-sfd-messages
+- ../../ffc-sfd-ahwp-stub
+- ../../ffc-sfd-data
+- ../../ffc-sfd-auth
+- ../../ffc-sfd-permissions-api
+- ../../ffc-sfd-messages-processor
```

Comment:
service "ffc-sfd-customer-receiver-payments" was not added to the list

Reply:
@ajaysoni-cap  @ppotturi  fyi